src/Products/urban/Extensions/fix_parceling.py

```
@@ -1,12 +1,37 @@
+
+def fix_changesDescription_field():
+    logger.info("Start fix changesDescription field")
+    brains = api.content.find(portal_type="Parcelling")
+    for brain in brains:
+        parcelling = brain.getObject()
+        if hasattr(parcelling.changesDescription, "raw"):
+            continue
+        description = parcelling.changesDescription
+        if description is None:
+            description = u"<p></p>"
+        if not isinstance(description, (str, unicode)):
```

Ruff (0.15.10)

[error] Undefined name `unicode`

(F821)